src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/restclient/java/MicroProfileRestClientDiagnosticsParticipant.java

```diff
@@ -118,7 +119,7 @@ private static void validateClassType(PsiClass classType, List<Diagnostic> diagn
 	private static void validateField(PsiField field, List<Diagnostic> diagnostics, JavaDiagnosticsContext context) {
 		String uri = context.getUri();
 		DocumentFormat documentFormat = context.getDocumentFormat();
-		boolean hasInjectAnnotation = AnnotationUtils.hasAnnotation(field, INJECT_ANNOTATION);
+		boolean hasInjectAnnotation = AnnotationUtils.hasAnnotation(field, INJECT_JAVAX_ANNOTATION) || AnnotationUtils.hasAnnotation(field, INJECT_JAKARTA_ANNOTATION);
```

Comment:
I think we should be better to change the signature of hasAnnotation with `String... annotationNames` to write

`AnnotationUtils.hasAnnotation(field, INJECT_ANNOTATION, INJECT_JAKARTA_ANNOTATION);` because:

 * it is more readable
 * we avoid to loop twice the IJ Annotation

LSP4MP should adopt this signature too.

Reply:
when you have `hasAnnotation(String... annotation)`, it's not clear without reading the javadoc, whether this means has ALL or has ANY of the passed annotations. So I'd rename the method to `hasAnyAnnotation`

Now do you want to do that as part of this PR or can it be done in a separate one?